migrations/20210506023749-add-budget-references.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    /**
+     * Add altering commands here.
+     *
+     * Example:
+     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+     */
+    await queryInterface.addConstraint('Budgets', {
+      fields: ['UserId'],
+      type: 'FOREIGN KEY',
+      references: {
+        table: 'Users',
+        field: 'id'
+      }
+    });
+    await queryInterface.addConstraint('Budgets', {
```

Comment:
This addConstraint call adds the UNIQUE constraint on UserId that enforces the ONE-TO-ONE relationship between User and Budget